doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -52,6 +52,17 @@ Build system
 
   * Manifest file entry ``mbedtls`` (:makevar:`ZEPHYR_MBEDTLS_MODULE_DIR`) checked out at path :file:`modules/crypto/mbedtls` now points to |NCS|'s fork of MbedTLS instead of Zephyr's fork.
 
+Board support
```

Comment:
The `pm-ext-flash` node was introduced before the previous release. I think we should not mention it in the commit message.

Reply:
@peknis, I did not found better category than that.
@kapi-no's and my changes were related to board configurations of nRF52 & nRF53 DKs and Thingy:53 so i thought it would be good idea to put it under `Board support`.
As i changed also app/dev guides i added entry at the bottom of a changelog with files that i edited.

Should i change that to something like `Working with nRF52 and nRF53 Series` then?

Reply:
Yes, use Working with nRF52 Series, Working with nRF53 and Application development as the headings for the changes. Look at the documentation structure on the left in  https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/index.html (the first level headings). Those are the choices for headings in the Changelog. I hope @divipillai and @annwoj agree with this. Talk with these ladies for immediate help.

Reply:
Yeah, agree with @peknis .